google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineTask.java

```diff
@@ -21,12 +21,20 @@
 /**
  * Encapsulates a task to be run on Google App Engine.
  */
-public interface AppEngineTask {
+public abstract class AppEngineTask {
```

Comment:
so that I can have an empty implementation of the onCancel method - I didn't want to force implementors to override this method if they didn't need it. 

It's a matter of taste I guess, but I prefer this to having a bunch of implementors with empty implementations.